frontend/src/pages/event.js

```diff
@@ -37,20 +41,44 @@ class Event extends Component {
         attendeeEmails: result.attendeeEmails
       })
 
-    const { result: res } = await getEventAttendees(query.id)
+    const { result: res, success } = await getEventAttendees(eventId)
+    if (success) {
+      this.setState({
+        attendees: res
+      })
+    }
+  }
+
+  toggleModal = () => {
     this.setState({
-      attendees: res != undefined ? res : []
+      modal: !this.state.modal
     })
   }
 
+  handleChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    })
+  }
+
+  addAttendee = async () => {
+    const attendee = {
+      name: this.state.attendeeName,
+      email: this.state.email,
+      year: this.state.year
+    }
+    const { success } = await eventCheckin(attendee, this.state.eventId)
+    return success
```

Comment:
Why is success returned? Doesn't look like the return value of addAttendee is used anywhere.

Reply:
it's used in the modal to display an error message if any fields are left blank